api/v1beta1/watcher_types.go

```diff
@@ -26,7 +26,7 @@ type WatcherSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	WatcherTemplate `json:",inline"`
```

Comment:
Now I doubt if this should be WatcherTemplate or WatcherCommon to define WatcherSpec.

If this is the spec of the top-level Watcher, shouldn't the template used by openstack-operator (WatcherTemplate) be the same defined in the WatcherSpec?

Reply:
indeed, I mixed up the two, fixed, thanks!